app/views/katello/api/v2/module_streams/_base.json.rabl

```diff
@@ -0,0 +1,8 @@
+object @resource
```

Comment:
docker_tags is using _base, but everything else is using 'base.json.rabl'.  Id rather not propagate the minority.  usually templates starting with _ indicate its a partial (meaning not used directly), but we use our base templates directly.